charts/authelia/README.md

```
@@ -3509,6 +3509,125 @@ false
 </td>
 			<td>Mounts an existing PersistentVolumeClaim.</td>
 		</tr>
+		<tr>
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)